src/main/java/appeng/client/gui/implementations/GuiCraftConfirm.java

```diff
@@ -432,6 +434,9 @@ private void drawListFG(final int offsetX, final int offsetY, final int mouseX,
 
                 if (stored != null && stored.getStackSize() > 0) {
                     lines++;
+                    if (stored.getUsedPercent() <= 100 && missingStack == null && pendingStack == null) {
```

Comment:
Is this needed?
```suggestion
                    if (missingStack == null && pendingStack == null) {
```

Reply:
For some reason, if you request a large craft (more than the integer limit) then even if there are missing components it doesn't show them as missing. The start button works correctly (it cannot be pressed because there are missing components). Examples on screenshots, tested on nightly 353 with the latest version of AE2. For this reason I put a check on percentages, because missing components are shown as stored can occur percentages greater than 100%. Maybe I should write issue about it.
![image](https://github.com/GTNewHorizons/Applied-Energistics-2-Unofficial/assets/136671315/eb73b00d-770c-4766-a176-8142dc1ef569)
![image](https://github.com/GTNewHorizons/Applied-Energistics-2-Unofficial/assets/136671315/0951e35b-8d16-4a04-bda8-1a78165cd56d)

Reply:
> For some reason, if request a large craft (more than the integer limit) then even if there are missing components it doesn't show them as missing. The start button works correctly (it cannot be pressed because there are missing components). Examples on screenshots, tested on nightly 353 with the latest version of AE2. For this reason I put a check on percentages, because missing components are shown as stored can occur percentages greater than 100%. Maybe I should write issue about it. ...

This might have been caused by the lack of `usedPercent` implementation in `AEFluidStack` if you tested it before that commit.

Reply:
> This might have been caused by the lack of `usedPercent` implementation in `AEFluidStack` if you tested it before that commit.

I've tested it before this pull request